modules/optics/src/main/scala/io/circe/optics/JsonObjectOptics.scala

```diff
@@ -22,6 +23,10 @@ trait JsonObjectOptics extends CatsConversions with ListInstances {
     }
   }
 
+  implicit final lazy val objectFoldKV: Fold[JsonObject, (String, Json)] = new Fold[JsonObject, (String, Json)] {
+    def foldMap[M: Monoid](f: ((String, Json)) => M)(obj: JsonObject): M = obj.toList.foldMap(f)
```

Comment:
This doesn't currently compile, and just adding the import won't work, since the cats syntax won't know what to do with the Scalaz monoid. I'd probably just write `scalaz.Foldable[List].foldMap(obj.toList)(f)`.